demos/common/mqtt_demo_helpers/mqtt_demo_helpers.h

```diff
@@ -88,7 +88,9 @@ BaseType_t EstablishMqttSession( MQTTContext_t * pxContext,
                                  MQTTEventCallback_t eventCallback );
 
 /**
- * @brief Handle the incoming packet if it's not related to the device shadow.
+ * @brief Invoked by the event callback to handle packet type being received.
+ * This updates the variable, #globalPacketTypeReceived, so that #prvWaitForPacket
```

Comment:
Isn't this an implementation detail? Both `globalPacketTypeReceived` and `prvWaitForPacket` are unknown to an application using this header file.